feature-libs/quote/components/summary/actions/quote-summary-actions.component.ts

```diff
@@ -173,141 +139,65 @@ export class QuoteSummaryActionsComponent
    * @protected
    */
   protected adjustBottomProperty(): void {
```

Comment:
@Platonn,
one thing that could / should be done is to find out a way how the action buttons stay before BottomHeaderSlot and not continue sliding above it.